src/libs/elektra/split.c

```diff
@@ -854,7 +854,7 @@ void splitCacheStoreState (KDB * handle, Split * split, KeySet * global, Key * p
 	ELEKTRA_LOG_DEBUG ("SIZE STORAGE STORE STUFF");
 	for (size_t i = 0; i < split->size; ++i)
 	{
-		// TODO: simplify this code below, seems like this affacts only the last split anyway
+		// TODO: simplify this code below, seems like this effects only the last split anyway
```

Comment:
This typo was meant to be "affects".

Reply:
Thank you! Seems like I got the wrong one in the 50/50 chance :smile: